saskatoon/sitebase/templates/app/detail_views/organization/equipment_table.html

```diff
@@ -0,0 +1,43 @@
+{% load i18n %}
+{% load static %}
+<div class="search-engine-area mg-t-30">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
+        <div class="data-table-list">
+          <div class="table-responsive">
+            <div>
+              <h4> {% trans Equipment %} </h4>
+              {% if perms.harvest.add_equipment and organization.is_equipment_point %}
```

Comment:
I think if this table is shown `organization.is_equipment_point` is already known to be `True`